### PR TITLE
Pin scipy to below 1.9.3

### PR DESCRIPTION
**Pull request by berland**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "numpy",
     "openpyxl",
     "pandas",
-    "scipy",
+    "scipy<1.9.3",
```
Comment (xjules) [inline setup.py:21]: Is it ok that we are not pinning scipy in ert?

Comment (berland) [reply setup.py:21]: Pyscal should be independent of ert in this regard. Ert should pin if it needs to.

**Review by hnformentin — COMMENTED** [review]
I think it would be nice to add a body in the commit message stating why scipy is being pinned. Then it would be easier if someone eventually consider upgrade it. 
What do you think about it?

**Comment by berland** [comment]
> I think it would be nice to add a body in the commit message stating why scipy is being pinned. Then it would be easier if someone eventually consider upgrade it. What do you think about it?

At least we should make an issue to unpin it. Not written anything specific in the commit message as I do not know the details of what has changed in scipy (not sure if it is a bug in scipy, or if pyscal has to adapt).

**Comment by hnformentin** [comment]
Nice suggestion. Please refer to this issue in the one to be created for future reference!

**Review by hnformentin — APPROVED** [review]

**Pull request merged**